frontend/admin/src/js/components/poolCandidate/PoolCandidatesTable.tsx

```diff
@@ -576,9 +559,7 @@ const PoolCandidatesTable: React.FC<{ poolId: string }> = ({ poolId }) => {
             description:
               "Text label for link to create new pool candidate on admin table",
           }),
-          path: getFeatureFlags().applicantSearch
-            ? adminRoutes.userCreate()
-            : adminRoutes.poolCandidateCreate(poolId),
+          path: adminRoutes.userCreate(),
```

Comment:
on that note, do we want to keep that button or change its function a bunch
now it renders a button that looks like this
![image](https://user-images.githubusercontent.com/40485260/206018231-c4c2d7ce-4d05-4849-8c95-4c8489128ee1.png)
but links to user creation form